packages/app/src/components/popup/tx-item.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import type {
+  Transaction,
+  MempoolTransaction,
+  TransactionEventFungibleAsset,
+} from '@blockstack/stacks-blockchain-api-types';
+import { Box, Flex, Text } from '@stacks/ui';
+import { truncateMiddle, stacksValue } from '@common/stacks-utils';
+import BigNumber from 'bignumber.js';
+
+type Tx = MempoolTransaction | Transaction;
+
+const getAssetTransfer = (tx: Tx): TransactionEventFungibleAsset | undefined => {
```

Comment:
Mixing between error case returning `null` or `undefined`. I'm a null man, myself. I've seen others advocating sticking to undefined. Either way, consistency 👍🏼